typescript/graphql-typegraphql-crud/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import 'reflect-metadata'
+import * as tq from 'type-graphql'
+import { ApolloServer } from 'apollo-server'
+import { createContext } from './context'
+import { resolvers } from "@generated/type-graphql";
+
+const app = async () => {
+  const schema = await tq.buildSchema({
+    resolvers,
+  })
+
+  const context = createContext()
+
+  new ApolloServer({ schema, context }).listen({ port: 4000 }, () =>
+    console.log(
+      `🚀 Server ready at: http://localhost:4000\n⭐️  See sample queries: http://pris.ly/e/ts/graphql-typegraphql-crud#using-the-graphql-api`,
```

Comment:
```suggestion
      `🚀 Server ready at: http://localhost:4000\n⭐️ See sample queries: http://pris.ly/e/ts/graphql-typegraphql-crud#using-the-graphql-api`,
```

Removing one space to clean up indentation:

![image](https://user-images.githubusercontent.com/4058327/107010743-09f97c80-6797-11eb-87b7-eec447d22d97.png)